_posts/2021-02-05-week-10.md

```diff
@@ -15,7 +15,7 @@ repronim_title: ReproMan – Execution and Environment Manager
 repronim_video: https://www.youtube.com/embed/grIVFbYH7YE
 repronim_slides:
 repronim_reading_links:
```

Comment:
```suggestion
repronim_reading_links: ["https://abcd-repronim.github.io/materials/week-10/"]
```
Link back to itself.

Reply:
Why link back to itself when you can have a [cat gif](https://media.giphy.com/media/1iu8uG2cjYFZS6wTxv/giphy.gif)?

Reply:
okay yes that's objectively better

Reply:
omg. how can I _not_ use that?